src/core/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FC } from "react";
+import LinkItem from "@/common/components/LInkItem/LinkItem";
+import Tag from "@/common/components/Tag/Tag";
+import "./Footer.scss";
+
+const tags = [
+  "Beans",
+  "Carrots",
+  "Apples",
+  "Garlic",
+  "Mushrooms",
+  "Tomatoes",
+  "Chilli peppers",
+  "Broccoli",
+  "Watermeloans",
+  "Oranges",
+  "Bananas",
+  "Grapes",
+  "Cherries",
+  "Meat",
+  "Seo tag",
+  "Fish",
+  "Seo tag",
+  "Freash food",
+  "Lemons",
+];
+
+const Footer: FC = () => {
+  return (
+    <footer className="footer">
+      <div className="footer__links">
+        <div className="footer__block">
+          <h3 className="footer__title">Get in touch</h3>
+          <LinkItem title="About Us" />
+          <LinkItem title="Careers" />
+          <LinkItem title="Press Releases" />
+          <LinkItem title="Blog" />
+        </div>
+        <div className="footer__block">
+          <h3 className="footer__title">Connections</h3>
+          <LinkItem title="Facebook" />
+          <LinkItem title="Twitter" />
+          <LinkItem title="Instagram" />
+          <LinkItem title="Youtube" />
+          <LinkItem title="Linkedin" />
+        </div>
+        <div className="footer__block">
+          <h3 className="footer__title">Earnings</h3>
+          <LinkItem title="Become an Affiliate" />
+          <LinkItem title="Advertise your product" />
+          <LinkItem title="Sell on Market" />
+        </div>
+        <div className="footer__block">
+          <h3 className="footer__title">Account</h3>
+          <LinkItem title="Your account" />
+          <LinkItem title="Returns Centre" />
+          <LinkItem title="100 % purchase protection" />
+          <LinkItem title="Chat with us" />
+          <LinkItem title="Help" />
+        </div>
```

Comment:
Think how to simplify this

Reply:
I create mock object like - { Account: ["Your account", "Returns Centre" ...], ... } and map throught it